CHANGELOG.md

```diff
@@ -39,7 +54,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 - `scalr_account_allowed_ips`: accept /32 suffix ([#224](https://github.com/Scalr/terraform-provider-scalr/pull/224))
-- `data.scalr_module_version`: if there are several module versions with the same version, select the version that has the 'is-root-module' flag set to true. ([#229](https://github.com/Scalr/terraform-provider-scalr/pull/229))
```

Comment:
I guess this was merged into [1.0.5] section by mistake after the v1.0.5 tag was created, so it's moving to [Unreleased] now, right?

Reply:
yes, https://scalr-labs.atlassian.net/browse/SCALRCORE-25897

Reply:
it will be release in 1.0.6